CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 2021.01.1
+
+**Released**: 2020.01.26
+
+**Commit Delta**: [Change from 2020.12.1 release](https://github.com/plus3it/spel/compare/2020.12.1...2021.01.1)
```

Comment:
```suggestion
**Commit Delta**: [Change from 2021.01.1 release](https://github.com/plus3it/spel/compare/2020.12.1...2021.01.1)
```

Reply:
This looks okay.  It's the manifests url that needs to be updated.

Reply:
Oops @eemperor is right. I meant to comment on the other line in my review, but think i forgot to "start a review", then submitted the request for change, then realized my comment was missing, then attempted to add the comment i meant to make, and apparently made it on the wrong line. So here we are.